tests/robustness/powerfailure_test.go

```diff
@@ -78,13 +78,49 @@ func TestRestartFromPowerFailure(t *testing.T) {
 		},
 	} {
 		t.Run(tc.name, func(t *testing.T) {
-			doPowerFailure(t, tc.du, tc.fsMountOpt, tc.useFailpoint)
+			doPowerFailure(t, tc.du, dmflakey.FSTypeEXT4, "", tc.fsMountOpt, tc.useFailpoint)
 		})
 	}
 }
 
-func doPowerFailure(t *testing.T, du time.Duration, fsMountOpt string, useFailpoint bool) {
-	flakey := initFlakeyDevice(t, strings.Replace(t.Name(), "/", "_", -1), dmflakey.FSTypeEXT4, fsMountOpt)
+func TestRestartFromPowerFailureXFS(t *testing.T) {
+	for _, tc := range []struct {
+		name         string
+		du           time.Duration
+		mkfsOpt      string
+		fsMountOpt   string
+		useFailpoint bool
+	}{
+		{
+			name:         "xfs_no_opts",
+			du:           5 * time.Second,
+			mkfsOpt:      "",
+			fsMountOpt:   "",
+			useFailpoint: true,
+		},
+		{
+			name:         "crc_off",
```

Comment:
this is probably silly to test for, because no sane person would run their partition with crc off, happy to remove that one again